cdap-docs/admin-manual/source/installation/packages.rst

```diff
@@ -16,49 +16,68 @@ Manual Installation using Packages
 
 This section describes installing CDAP on Hadoop clusters that are:
 
-- Generic Apache Hadoop distributions; 
+- Generic Apache Hadoop distributions;
 - CDH (Cloudera Distribution of Apache Hadoop) clusters *not managed* with Cloudera Manager; or
 - HDP (Hortonworks Data Platform) clusters *not managed* with Apache Ambari.
 
 Cloudera Manager (CDH), Apache Ambari (HDP), and MapR distributions should be installed
 with our other :ref:`distribution instructions <installation-index>`.
 
 - As CDAP depends on HDFS, YARN, HBase, ZooKeeper, and (optionally) Hive and Spark, it must be installed
-  on cluster host(s) with full client configurations for these dependent services. 
+  on cluster host(s) with full client configurations for these dependent services.
 
 - The CDAP Master Service must be co-located on a cluster host with an HDFS client, a YARN
   client, an HBase client, and |---| optionally |---| Hive or Spark clients.
 
-- Note that these clients are redundant if you are co-locating the CDAP Master  
+- Note that these clients are redundant if you are co-locating the CDAP Master
   on a cluster host (or hosts, in the case of a deployment with high availability) with
-  actual services, such as the HDFS Namenode, the YARN resource manager, or the HBase
+  actual services, such as the HDFS NameNode, the YARN resource manager, or the HBase
   Master.
-  
-- You can download the `Hadoop client <http://hadoop.apache.org/releases.html#Download>`__ 
+
+- You can download the `Hadoop client <http://hadoop.apache.org/releases.html#Download>`__
   and `HBase client <http://www.apache.org/dyn/closer.cgi/hbase/>`__ libraries, and then
   install them on the hosts running CDAP services. No Hadoop or HBase services need be running.
 
-- All services run as the ``'cdap'`` user installed by the package manager.
+- All services run as the ``cdap`` user installed by the package manager.
 
 - If you are installing CDAP with the intention of using *replication,* see these
   instructions on :ref:`CDAP Replication <installation-replication>` *before* installing or starting CDAP.
 
 
 Preparing the Cluster
 =============================
-Please review the :ref:`Software Prerequisites <admin-manual-software-requirements>`, 
+Please review the :ref:`Software Prerequisites <admin-manual-software-requirements>`,
 as a configured Hadoop, HBase, and Hive (plus an optional Spark client) needs to be configured on the
-node(s) where CDAP will run. 
+node(s) where CDAP will run.
 
 .. Hadoop Configuration
 .. --------------------
 .. include:: ../_includes/installation/hadoop-configuration.txt
 
+Create the "cdap" User
```

Comment:
I think we need to preface this section with the fact that the base cdap package will install cdap as a system user. 
 However, for the reasons of consistent UIDs across the cluster, or ensuring it is created on the namenode when installing cdap elsewhere, you need to make sure the cdap user is installed consistently on all nodes of the cluster.

Users should be able to ascertain that the base package will install the cdap user automatically, and that is still an option for them (if on singlenode cluster, etc).